src/process_utility.c

```diff
@@ -183,13 +170,24 @@ relation_not_only(RangeVar *rv)
 				 errmsg("ONLY option not supported on hypertable operations")));
 }
 
+static void
+process_add_hypertable(ProcessUtilityArgs *args, Hypertable *ht)
+{
+	args->hypertable_list = lappend(args->hypertable_list, ht);
+}
+
+static void
+process_add_subcmd(ProcessUtilityArgs *args, void *datum)
+{
+	args->subcmd_list = lappend(args->subcmd_list, datum);
```

Comment:
so this subcmd_list doesn't have any ordering guarantees with regard to `args->hypertable_list` do we need it to?